.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,34 +1,17 @@
-<!--
-See http://loopback.io/doc/en/contrib/triaging-pull-requests.html to understand
-the MINIMUM requirements for submitting pull requests.
--->
+### Description
 
-### Type of change
-
-<!--
-See http://semver.org/ for more info.
--->
-
-- [ ] SEMVER-MAJOR -- Backwards-incompatible changes
-- [ ] SEMVER-MINOR -- Add functionality in a backwards-compatible manner
-- [ ] SEMVER-PATCH -- Backwards-compatible bug fixes
```

Comment:
@bajtos There was a proposal during demo to remove this as we should be the ones determining this to (instead of putting the burden on community -- we determine what are patches, backwards compatible etc). Also remove extra barriers for contributors and I agree -- as did @raymondfeng @ritch during demo.